ocrd_anybaseocr/cli/ocrd_anybaseocr_block_segmentation.py

```diff
@@ -77,9 +92,9 @@ def process(self):
                 model can be downloaded from http://url
                 """ % model_path)
             sys.exit(1)
-            
+        '''    
         config = InferenceConfig()
-        mrcnn_model = model.MaskRCNN(mode="inference", model_dir=str(model_path), config=config)
+        mrcnn_model = model.MaskRCNN(mode="inference", model_dir="./", config=config)
```

Comment:
On first sight, this does not seem to be a good idea. You should keep the possibility to set model paths via a parameter.

Reply:
Done.